malariagen_data/anopheles.py

```diff
@@ -2269,6 +2273,65 @@ def cohort_diversity_stats(
 
         return pd.Series(stats)
 
+    # @staticmethod?
+    def _setup_cohorts(
+        self,
+        cohorts: base_params.cohorts,
+        sample_sets: Optional[base_params.sample_sets] = None,
+        sample_query: Optional[base_params.sample_query] = None,
+        cohort_size: Optional[base_params.cohort_size] = 15,
+        min_cohort_size: Optional[base_params.min_cohort_size] = None,
+    ):
```

Comment:
```suggestion
    def _setup_cohorts(
        self,
        cohorts: base_params.cohorts,
        sample_sets: Optional[base_params.sample_sets],
        sample_query: Optional[base_params.sample_query],
        cohort_size: Optional[base_params.cohort_size],
        min_cohort_size: Optional[base_params.min_cohort_size],
    ):
```

Because this is a private method, default values aren't needed here. Default values will be provided in the public method which calls this internally.